### PR TITLE
PROD-97: contact page

### PR DESCRIPTION
**Pull request by shevchukm**
# Reviewers
- [ ] @Antoniozinchenko
- [ ] @okohuch 
- [ ] @VitaliiChverenko 
# Description

add contacts page, edit modal windows, google map

# Checklist:

- [ ] PR is approved by 1 team mate
- [ ] PR is approved by 2 experts

### CODE REVIEW

File: src/components/contacts/AddressAutoComplete.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        coords:{
+          lat: place.geometry.location.lat(),
+          lng: place.geometry.location.lng() 
+        }
+      }
+      props.onSelectAddress(address);
+    }}
+    placeholder={props.address}
+    types={['address']}/>
+    </div>
```
Comment (VitaliiChverenko) [inline src/components/contacts/AddressAutoComplete.js]: Open tag has to be over the end tag

Comment (VitaliiChverenko) [reply src/components/contacts/AddressAutoComplete.js]: Use return() for multiple lines

File: src/components/contacts/AddressAutoComplete.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+export default GoogleApiWrapper({
+    apiKey: GoogleAPIkey,
+    libraries: ['places']
+  })(AddressAutoComplete)
```
Comment (VitaliiChverenko) [inline src/components/contacts/AddressAutoComplete.js]: Add the line

File: src/components/contacts/ContactForm.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            }
+        };
+        this.onSubmit = this.onSubmit.bind(this);
+        this.handleOpen = this.handleOpen.bind(this);
+        this.handleValue = this.handleValue.bind(this);
+        this.handleClose = this.handleClose.bind(this);
+    }
+
+    componentWillReceiveProps(props) {
+        this.setState({contacts: {...props.contactsData}});
```
Comment (VitaliiChverenko) [inline src/components/contacts/ContactForm.js]: too many spaces before this.setState

File: src/components/contacts/AddressAutoComplete.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {GoogleApiWrapper} from 'google-maps-react';
+import Autocomplete from 'react-google-autocomplete';
+import GoogleAPIkey from '../../constants/google.config'
+import './AddressAutoComplete.css';
+
+const AddressAutoComplete = props =>{
+  return (
+  <div className='ui input address-autocomplete labeled'>
+  <div class="ui label label">Address</div>
```
Comment (Antoniozinchenko) [inline src/components/contacts/AddressAutoComplete.js]: use className instead of class

**Review by Antoniozinchenko — CHANGES_REQUESTED** [review]

File: src/components/contacts/AddressAutoComplete.js (hunk 4 above)
Comment (shevchukm) [reply src/components/contacts/AddressAutoComplete.js]: done

**Review by Antoniozinchenko — APPROVED** [review]

**Pull request merged**